tsconfig.js.json

```diff
@@ -4,4 +4,11 @@
 		"noImplicitAny": false,
 		"checkJs": true
 	},
+	"include": [
+		"src/**/*",
+		"test/**/*",
+		".eslintrc.js",
+		"*.ts",
+		"*.js"
```

Comment:
Are these two lines necessary?

Reply:
We need the `*.js` for the `.eslintrc.js`, but the `*.ts` was for future proofing tbh, let me remove it.